src/export_edf.jl

```diff
@@ -155,6 +155,11 @@ function reencode_samples(samples::Samples, sample_type::Type{<:Integer}=Int16)
     # maximize the dynamic range of Int16 encoding.
     samples = decode(samples)
     smin, smax = extrema(samples.data)
+    # If the input is flat, normalize to zero and one
+    if smin == smax
+        smax = one(smax)
+        smin = zero(smin)
+    end
```

Comment:
How about this?  then we actually get the correct value out...

```suggestion
    # If the input is flat, normalize to zero and one
    if smin == smax
        smax = smin + one(smax)
    end
```